### PR TITLE
feat: add quickstart akamai-datastream2

### PR DESCRIPTION
**Pull request by gayatrisiva**
# Summary

Submitting request for new quickstart for akamai-datastream2. Changes added:

- added dashboard json
- added config.yml with descriptions
- added dashboard screenshots
- added baseline alert definition

## Pre merge checklist

- [x] Did you check your NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

datastream2-01
datastream2-02
datastream2-03

### CODE REVIEW

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Thanks for your contribution, @gayatrisiva! This looks pretty much good to go, I just noticed one field that needs a name change.

File: quickstarts/datastream2-akamai/config.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Name of the quickstart (required)
+# sets the URL name of the quickstart on public I/O
+name: datastream2
```
[the remark this answers is not in the document]
Comment (gayatrisiva) [reply quickstarts/datastream2-akamai/config.yml]: Updated.

File: quickstarts/datastream2-akamai/dashboards/datastream2/datastream2.json [hunk 2]
```diff
@@ -0,0 +1,632 @@
+{
+  "name": "Akamai Datastream2",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
```
[the remark this answers is not in the document]
Comment (gayatrisiva) [reply quickstarts/datastream2-akamai/dashboards/datastream2/datastream2.json]: Updated.

File: quickstarts/datastream2-akamai/config.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  - logs
+  - cdn
+  - datastream
+  - performance
+  - security
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - third-party-datastream
```
[the remark this answers is not in the document]
Comment (gayatrisiva) [reply quickstarts/datastream2-akamai/config.yml]: @aswanson-nr The install plan is currently a WIP from our end. We should have this ready for uploading logs to NewRelic soon and I shall update the link for the same when it is available.

[the remark this answers is not in the document]
Comment (jcountsNR) [reply quickstarts/datastream2-akamai/config.yml]: @gayatrisiva , Sorry sounds like there is some confusion here. There is an install plan, and a data source which I believe you are confusing. The data source can be enhanced later, but an install plan
1. Install Plan - YML file that instructs I/O how to install your quickstart (housed in the install/third-party directory). You can look at the other files in the install/third-party dir for reference.
2. Data Source - a doc file guiding an end user on how to integrate your platform data with NR. We will probably build a doc file for this on our end once the quickstart is live.

**Review by aswanson-nr — APPROVED** [review]
@gayatrisiva, this looks good to go to me! We just need you to sign the Contributor License Agreement before we merge. Thanks for your contribution!

**Pull request merged**